server/controller/socialMediaController.js

```diff
@@ -0,0 +1,52 @@
+import Authenticate from '../middleware/auth/Authenticate';
+import models from '../models';
+
+const { User } = models;
+const socialMediaCallback = async (accessToken, refreshToken, profile, done) => {
```

Comment:
`socialSignOn` instead of `socialMediaCallback`

Reply:
resolved